test/connectors/NPSConnectorTest.scala

```diff
@@ -131,5 +131,21 @@ class NPSConnectorTest extends BaseSpec with WireMockHelper with ScalaFutures wi
 
       result.status shouldBe INTERNAL_SERVER_ERROR
     }
+
+    "return INTERNAL_SERVER_ERROR response in case of 503 from NPS" in new NPSConnectorSetUp {
+
+      val url = s"/individuals/annual-tax-summary/" + testNinoWithoutSuffix + "/" + invalidTaxYear
```

Comment:
Does this API return a 503? Have you checked the API docs to see if there are any other statuses we may have missed?

Reply:
Yes it can return 503. Checked API docs and all possible responses have been tested.